web/docs/about-vast/why-vast.md

```diff
@@ -0,0 +1,205 @@
+operations, for the following reasons:
+
+- **Vertical Scaling**: when co-locating a storage engine next to high-volume
+  data sources, e.g., on a network appliance together with a network monitor,
+  CPU and memory constraints coupled with a non-negligible IPS overhead prohibit
+  scaling horizontally to build a "cluster in a box."
+
+- **Analytical Workloads**: the document-oriented storage does not perform well
+  for analytical workloads, such as group-by and aggregation queries. But such
+  analytics are very common in interactive threat hunting scenarios and in
+  various threshold-based detections. VAST leverages Arrow for columnar data
+  representation and partially for query execution.
+
+- **Economy of Representation**: security telemetry data exhibits a lot of
+  repetitiveness between events, such as similar IP addresses, URL prefixes, or
+  log message formats. This data compresses much better when transposed into a
+  columnar format, such as Parquet.
+
+A special case of document DBs are full-text search engines, such as
+ElasticSearch or Solr. The unit of input is typically unstructured text. The
+search engine uses (inverted) indexes and ranking methods to return the most
+relevant results for a given combination of search terms.
+
+:::note Recommendation
+Most of the security telemetry arrives as structured log/event data, as opposed
+to unstructured textual data. If your primary use case involves working with
+text, VAST might not be a good fit. That said, needle-in-haystack search
+and other information retrieval techniques are still relevant for security
+analytics, for which VAST has basic support.
+:::
+
+## VAST vs. Timeseries DBs
+
+Timeseries databases share a lot in common with [OLAP
+engines](#vast-vs-data-warehouses), but put center data organization around
+time.
+
+:::note Recommendation
+If you plan to access your event data through time domain and need to model the
+majority of data as series, a timeseries DBs may suit the bill. If you access
+data through other (spatial) attributes, like IP addresses or domains, a
+traditional timeseries DB might not be good fit—especially for high-cardinality
+attributes. If your analysis involve running more complex detections, or
+include needle-in-haystack searches, VAST might be a better fit.
+:::
+
+## VAST vs. Key-Value DBs
+
+A key-value store performs a key-based point or range lookup to retrieve one or
+more values. Security telemetry is high-dimensional data and there are many more
+desired entry points than a single key besides time, e.g., IP address,
+application protocol, domain name, or hash value.
+
+:::note Recommendation
+Key-value stores alone are not suitable as foundation for running security
+analytics workloads. There are narrow use cases where key-value stores can
+facilitate certain capabilities, e.g., when processing watch lists. (VAST offers
+a *matcher* plugin for this purpose.)
```

Comment:
Do we have a page to link to already for this?

Reply:
Not yet, we'll sprinkle links over this page later.